### PR TITLE
Updated description for `hasErrors` input

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-docs-update](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-docs-update)

File: src/app/public/modules/input-box/input-box.component.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ import {
 export class SkyInputBoxComponent implements OnInit {
 
   /**
-   * Indicates whether to visually highlight the input box in an error state. The input box also displays
+   * Indicates whether to visually highlight the input box in an error state. If not provided, the input box displays
```
Comment (blackbaud-johnly) [inline src/app/public/modules/input-box/input-box.component.ts:48]: I'm not really sure that `If not provided` is meaningful for a Boolean property. I'm making a couple suggested tweaks to the docs to try to make this more clear at a glance...
```suggestion
   * Indicates whether to visually highlight the input box in an error state. If `hasErrors` is `false`, the input box still
```

File: src/app/public/modules/input-box/input-box.component.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ import {
...
    * an error state if the `ngModel` or Angular `FormControl` contains an error.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/input-box/input-box.component.ts:49]:
```suggestion
   * displays in an error state when either the `ngModel` or the Angular `FormControl` contain an error.
```

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-docs-update](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-docs-update)

**Pull request merged**